clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/PassiveReplicationMessage.java

```diff
@@ -186,4 +189,23 @@ public long getKey() {
       return key;
     }
   }
+
+  public static class ServerStoreLifeCycleReplicationMessage extends ClientIDTrackerMessage {
+
+    private final LifecycleMessage message;
```

Comment:
Really not too fond of the wrap a message in a message approach. It seems lazy 😉